Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1765,19 +1765,29 @@ namespace Remotion.BocAutoCompleteReferenceValue
         if (!isVisibe)
         {
             popUp.style.width = 'auto'; // clear the width before showing the popUp, otherwise, the popUp expands to 100%
+            popUp.style.setProperty("scrollbar-gutter", "stable") //Typescript build error when using 'popUp.style.scrollbarGutter = "stable";'
```

Comment:
Oh, yeah, I just realized, we should add backup for this and restore the backup further down when resetting the value. otherwise, a future css theme  could introduce a scrollbar-gutter and get the look-and-feel destroyed here. No, it's not likely, but it's still good practice to avoid such tripwires for future-self.